src/test/java/org/reaktivity/nukleus/socks/internal/ConnectionIT.java

```diff
@@ -0,0 +1,222 @@
+    public void shouldConnectedIpv6() throws Exception
+    {
+        k3po.finish();
+    }
+
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${forward}/connected.then.client.abort/server",
+        "${client}/rfc1928/connect/connected.then.client.abort/clint"
+    })
+    public void shouldConnectedThenClientAbort() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${forward}/connected.then.client.close/server",
+        "${client}/rfc1928/connect/connected.then.client.close/client"
+    })
+    public void shouldConnectedThenClientClose() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${forward}/connected.then.client.reset/server",
+        "${client}/rfc1928/connect/connected.then.client.reset/client"
+    })
+    public void shouldConnectedThenClientReset() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${forward}/connected.then.client.write.data/server",
+        "${client}/rfc1928/connect/connected.then.client.write.data/client"
+    })
+    public void shouldConnectedThenClientWriteData() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+            "${route}/server/controller",
+            "${forward}/connected.then.server.abort/server",
+            "${client}/rfc1928/connect/connected.then.server.abort/client"
+    })
+    public void shouldConnectedThenServerAbort() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+            "${route}/server/controller",
+            "${forward}/connected.then.server.close/server",
+            "${client}/rfc1928/connect/connected.then.server.close/client"
+    })
+    public void shouldConnectedThenServerClose() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+            "${route}/server/controller",
+            "${forward}/connected.then.server.reset/server",
+            "${client}/rfc1928/connect/connected.then.server.reset/client"
+    })
+    public void shouldConnectedThenServerReset() throws Exception
+    {
+        k3po.finish();
+    }
+
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${forward}/connected.then.server.write.data/server",
+        "${client}/rfc1928/connect/connected.then.server.write.data/client"
+    })
+    public void shouldConnectedThenServerWriteData() throws Exception
+    {
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        //"${forward}/rejected.address.type.not.supported/server",
+        "${client}/rfc1928/connect/rejected.address.type.not.supported/client"
+    })
+    public void shouldRejectedAddressTypeNotSupported() throws Exception
```

Comment:
Please rename all remaining methods called `shouldRejected...` to `shouldReject...`.